testing/run_travis_tests.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash -e
+
+SCRIPT_DIR=$( cd $( dirname "$0" ) && pwd )
+
+. "${SCRIPT_DIR}/testing_lib.sh"
+
+IS_PULL_REQUEST=
+if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
+  IS_PULL_REQUEST=1
+fi
+
+BRANCH="$TRAVIS_BRANCH"
+
+if [ -z "$IS_PULL_REQUEST" ] && [ "$BRANCH" = "master" ]; then
+  # Test all images on master
+  smoke_test_all_images
+  smoke_test_non_root
```

Comment:
It seems there is something wrong with this PR because it is not taking the good branch. I cannot find the non-root execution in the [travis build logs](https://travis-ci.org/docker-flink/docker-flink/builds/477953909). Can you please check.

Reply:
`smoke_test_all_images` and `smoke_test_non_root` only runs on the master branch for non-PRs; for PRs, only `smoke_test_one_image` is run (second half of this if-statement).

Reply:
Yes I understood that, it was just curious for me that the PR that fixed support for test_non_root did not test non root. But well it will be tested anyway so ok.

Reply:
Ah yeah, hah. It's just that these tests are pretty heavyweight and I figure just running them on the master branch is okay. I could be convinced otherwise in the case of the non_root tests, running them for PRs as well—feel free to open another PR to switch things up if you want to do that.